whois-api/src/test/java/net/ripe/db/whois/api/zonemasterservice/ZonemasterDummy.java

```diff
@@ -1,11 +1,14 @@
-package net.ripe.db.whois.api;
+package net.ripe.db.whois.api.zonemasterservice;
 
 import com.google.common.collect.Lists;
 import com.google.common.collect.Maps;
+import com.hazelcast.internal.json.Json;
+import com.hazelcast.internal.json.JsonObject;
 import net.ripe.db.whois.common.Stub;
```

Comment:
Can you check this import? We should not be using json object from hazelcast library

Reply:
removed